src/test/java/io/javalin/TestSSE.java

```diff
@@ -0,0 +1,73 @@
+package io.javalin;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.Unirest;
+import io.javalin.serversentevent.EventSource;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Future;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestSSE {
+    final String event = "hi";
+    final String data = "hello world";
+    final String ssePath = "/sse";
+    Javalin server;
+    int port;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
You can use the `TestUtil.test()` method, it takes care of starting/stopping the server.